applications/app/views/fragments/newGalleryBody.scala.html

```diff
@@ -0,0 +1,124 @@
+@(gallery: model.Gallery, related: model.RelatedContent, index: Int)(implicit request: RequestHeader)
+@import common.{Edition, LinkTo}
+@import layout.ContentWidths.GalleryMedia
+@import views.support.TrailCssClasses.toneClass
+@import views.support.{ImgSrc, RenderClasses}
+
+
+@*
+
+TODO:
```

Comment:
I left a lot of todos in the code for future iteration. Apologies in advance if it is confusing!